src/components/TitleBar/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { DownloadBtn, DownloadBtnProps } from './DownloadBtn';
+import { Colors } from '../../constants';
+
+/**
+ * interface for TitleBar
+ * @export
+ * @interface TitleBarProps
+ */
+export interface TitleBarProps {
+  title: string;
+  desc?: string;
+  downloadBtnProps?: DownloadBtnProps;
+}
+
+/**
+ * Renders a TitleBar that shows a title, a description, a download button.
+ * The description and the download button are optional although the title is always shown
+ * @export
+ * @function TitleBar
+ * @param {string} title
```

Comment:
We need to add the target and target description to the Title bar. For instance if we are viewing ELA Grade 3 Claim 1 Target 3 the title bar should have the following content:
Claim 1: Students can read closely and analytically to comprehend a range of increasingly complex literary and informational texts.
Target 3: WORD MEANINGS: Determine intended meanings of words, including words with multiple meanings (academic/tier 2 words), based on context,word relationships, word structure (e.g., common roots, affixes), or use of reference materials (e.g., beginning dictionary), with primary focus on determining meaning based on context and the academic (tier 2) vocabulary common to complex texts in all disciplines. 

This is probably going to make the title bar a bit wider or taller but that is ok.

Reply:
I have added the target and target description into the title bar component.
Please review below commits.